### PR TITLE
Fix main

### PR DESCRIPTION
**Pull request by TwistedHead**
This is ready for review.

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,226 +1,207 @@
...
+*ITC205 Asg2 - Group7 (Jasemine, Ian, Meraj)
+*/
+
+public class Main {
+
+	private static Scanner IN;
+	private static library LIB;
+	private static String MENU;
+	private static Calendar CAL;
+	private static SimpleDateFormat SDF;
```
Comment (jazzie-b) [inline src/Main.java]: All variables above should be changed so something more meaningful. They should also follow camel back case

File: src/Main.java [hunk 2]
```diff
@@ -1,226 +1,207 @@
...
+
+		return stringBuilder.toString();
+	}
+
+	public static void main(String[] args) {
+		try {
+			IN = new Scanner(System.in);
+			LIB = library.INSTANCE();
+			CAL = Calendar.getInstance();
+			SDF = new SimpleDateFormat("dd/MM/yyyy");
```
Comment (jazzie-b) [inline src/Main.java]: Indentation in block statements should be 2 spaces not a tab

File: src/Main.java [hunk 3]
```diff
@@ -1,226 +1,207 @@
...
-}
+import java.text.SimpleDateFormat;
+import java.util.Scanner;
+
+/*
+*ITC205 Asg2 - Group7 (Jasemine, Ian, Meraj)
+*/
+
+public class Main {
+
```
Comment (jazzie-b) [inline src/Main.java]: Tabs used for indenting instead of spaces.

File: src/Main.java [hunk 4]
```diff
@@ -1,226 +1,207 @@
...
+		output("\n" + book + "\n");
+
+	}
+
+	private static void addMember() {
+		try {
+			String lastName = input("Enter last name: ");
+			String firstName = input("Enter first name: ");
+			String email = input("Enter email: ");
+			int phoneNo = Integer.valueOf(input("Enter phone number: ")).intValue();
```
Comment (jazzie-b) [inline src/Main.java]: phoneNo could be changed to something more meaningful

File: src/Main.java [hunk 5]
```diff
@@ -1,226 +1,207 @@
...
+			member member = LIB.Add_mem(lastName, firstName, email, phoneNo);
+			output("\n" + member + "\n");
+
+		} catch (NumberFormatException e) {
+			output("\nInvalid phone number\n");
+		}
+
+	}
+
+	private static String input(String prompt) {
```
Comment (jazzie-b) [inline src/Main.java]: Method name could be changed to something more meaningful

File: src/Main.java [hunk 6]
```diff
@@ -1,226 +1,207 @@
...
+				.append("  P  : Pay fine\n")
+				.append("\n")
+				.append("  T  : Increment date\n")
+				.append("  Q  : Quit\n")
+				.append("\n")
+				.append("Please enter you choice : ");
+
+		return stringBuilder.toString();
+	}
+
```
Comment (jazzie-b) [inline src/Main.java]: Should be 2 line spacing between methods, issue throughout

File: src/Main.java [hunk 7]
```diff
@@ -1,226 +1,207 @@
...
+		}
+
+	}
+
+	private static String input(String prompt) {
+		System.out.print(prompt);
+		return IN.nextLine();
+	}
+
+	private static void output(Object object) {
```
Comment (jazzie-b) [inline src/Main.java]: Method name could be changed to something more meaningful.

File: src/Main.java (hunk 6 above)
Comment (TwistedHead) [reply src/Main.java]: Updated as requested.

File: src/Main.java (hunk 7 above)
Comment (TwistedHead) [reply src/Main.java]: Updated as requested.

File: src/Main.java [hunk 8]
```diff
@@ -1,226 +1,207 @@
...
-		System.out.println(object);
-	}
-
-	
-}
+import java.text.SimpleDateFormat;
+import java.util.Scanner;
+
+/*
+*ITC205 Asg2 - Group7 (Jasemine, Ian, Meraj)
```
[the remark this answers is not in the document]
Comment (TwistedHead) [reply src/Main.java]: Updated name.

File: src/Main.java (hunk 1 above)
Comment (TwistedHead) [reply src/Main.java]: Updated variables name.

File: src/Main.java (hunk 2 above)
Comment (TwistedHead) [reply src/Main.java]: updated with recommended block spacing

File: src/Main.java (hunk 3 above)
Comment (TwistedHead) [reply src/Main.java]: updated as per suggestion.

File: src/Main.java (hunk 4 above)
Comment (TwistedHead) [reply src/Main.java]: Unnecessary update requested. phoneNo is a meaningful name.

File: src/Main.java [hunk 9]
```diff
@@ -1,226 +1,222 @@
...
+				.append("  T  : Increment date\n")
+				.append("  Q  : Quit\n")
+				.append("\n")
+				.append("Please enter you choice : ");
+
+		return stringBuilder.toString();
+	}
+
+	
+	public static void main(String[] args) {
```
Comment (jazzie-b) [inline src/Main.java]: Main method should be moved to correct position. see style guide.

File: src/Main.java [hunk 10]
```diff
@@ -1,226 +1,222 @@
...
+				memberOutput(book);
+			}
+
+			menu = getMenu();
+
+			boolean userSelection = false;
+
+			while (!userSelection) {
+
+				memberOutput("\n" + simpleDateFormat.format(calender.Date()));
```
Comment (jazzie-b) [inline src/Main.java]: Remove method in method calls, see style guide.

File: src/Main.java [hunk 11]
```diff
@@ -1,226 +1,222 @@
...
+			}
+		} catch (RuntimeException e) {
+			memberOutput(e);
+		}
+		memberOutput("\nEnded\n");
+	}
+
+	
+	private static void payFine() {
+		new PayFineUi(new PayFineControl()).run();
```
Comment (jazzie-b) [inline src/Main.java]: Remove method from method call, see style guide.

File: src/Main.java [hunk 12]
```diff
@@ -1,226 +1,222 @@
...
+	private static void listMembers() {
+		memberOutput("");
+		for (Member member : library.members()) {
+			memberOutput(member + "\n");
+		}
+	}
+
+	
+	private static void borrowBook() {
+		new BorrowBookUi(new BorrowBookControl()).run();
```
Comment (jazzie-b) [inline src/Main.java]: Remove method from method call, see style guide.

File: src/Main.java [hunk 13]
```diff
@@ -1,226 +1,222 @@
...
+	}
+
+	
+	private static void returnBook() {
+		new ReturnBookUi(new ReturnBookControl()).run();
+	}
+
+	
+	private static void fixBooks() {
+		new FixBookUi(new FixBookControl()).run();
```
Comment (jazzie-b) [inline src/Main.java]: Remove method from method call, see style guide.

File: src/Main.java [hunk 14]
```diff
@@ -1,226 +1,222 @@
...
+	}
+
+	
+	private static void borrowBook() {
+		new BorrowBookUi(new BorrowBookControl()).run();
+	}
+
+	
+	private static void returnBook() {
+		new ReturnBookUi(new ReturnBookControl()).run();
```
Comment (jazzie-b) [inline src/Main.java]: Remove method from method call, see style guide.

File: src/Main.java [hunk 15]
```diff
@@ -1,226 +1,222 @@
...
+
+	
+	private static void fixBooks() {
+		new FixBookUi(new FixBookControl()).run();
+	}
+
+	
+	private static void incrementDate() {
+		try {
+			int days = Integer.valueOf(memberInput("Enter number of days: ")).intValue();
```
Comment (jazzie-b) [inline src/Main.java]: Remove method from method call, see style guide.

File: src/Main.java [hunk 16]
```diff
@@ -1,226 +1,222 @@
...
+			calender.incrementDate(days);
+			library.checkCurrentLoans();
+			memberOutput(simpleDateFormat.format(calender.Date()));
```
Comment (jazzie-b) [inline src/Main.java]: Remove method from method call, see style guide.

File: src/Main.java [hunk 17]
```diff
@@ -1,226 +1,222 @@
...
+
+	}
+
+	
+	private static void addMember() {
+		try {
+			String lastName = memberInput("Enter last name: ");
+			String firstName = memberInput("Enter first name: ");
+			String email = memberInput("Enter email: ");
+			int phoneNo = Integer.valueOf(memberInput("Enter phone number: ")).intValue();
```
Comment (jazzie-b) [inline src/Main.java]: Remove method from method call, see style guide.

File: src/Main.java [hunk 18]
```diff
@@ -1,226 +1,222 @@
...
+			memberOutput(simpleDateFormat.format(calender.Date()));
+
+		} catch (NumberFormatException e) {
+			memberOutput("\nInvalid number of days\n");
+		}
+	}
+
+	
+	private static void addBook() {
+
```
Comment (jazzie-b) [inline src/Main.java]: Unnecessary line spacing

File: src/Main.java [hunk 19]
```diff
@@ -1,226 +1,222 @@
...
-	
-	private static void output(Object object) {
-		System.out.println(object);
-	}
-
-	
-}
+import java.text.SimpleDateFormat;
+import java.util.Scanner;
+
```
Comment (jazzie-b) [inline src/Main.java]: Tabs used instead of spacing, see style guide for reference

File: src/Main.java (hunk 11 above)
Comment (TwistedHead) [reply src/Main.java]: Unable to update as this is beyond my skill level and not a crucial requirement for this assignment.

File: src/Main.java (hunk 12 above)
Comment (TwistedHead) [reply src/Main.java]: Unable to update as this is beyond my skill level and not a crucial requirement for this assignment.

File: src/Main.java (hunk 13 above)
Comment (TwistedHead) [reply src/Main.java]: Unable to update as this is beyond my skill level and not a crucial requirement for this assignment.

File: src/Main.java (hunk 14 above)
Comment (TwistedHead) [reply src/Main.java]: Unable to update as this is beyond my skill level and not a crucial requirement for this assignment.

File: src/Main.java (hunk 15 above)
Comment (TwistedHead) [reply src/Main.java]: Unable to update as this is beyond my skill level and not a crucial requirement for this assignment.

File: src/Main.java (hunk 17 above)
Comment (TwistedHead) [reply src/Main.java]: Unable to update as this is beyond my skill level and not a crucial requirement for this assignment.

File: src/Main.java (hunk 19 above)
Comment (TwistedHead) [reply src/Main.java]: updated

File: src/Main.java (hunk 9 above)
Comment (TwistedHead) [reply src/Main.java]: updated

File: src/Main.java (hunk 19 above)
Comment (jazzie-b) [reply src/Main.java]: Not updated, still tabs

File: src/Main.java (hunk 17 above)
Comment (jazzie-b) [reply src/Main.java]: Not fixed.

**Pull request merged**